### DISCUSSION TITLE
v0.5.0

### DISCUSSION THREAD

**Discussion by jackmoody11** [sec-edgar/sec-edgar#287] [General]
Adding functions to easily access the SEC's new REST API! You can read more about the API from the SEC [here](https://www.sec.gov/edgar/sec-api-documentation). The documentation for secedgar on the new rest.py module is [here](https://sec-edgar.github.io/sec-edgar/rest_api.html).

<hr /><em>This discussion was created from the release <a href='https://github.com/sec-edgar/sec-edgar/releases/tag/v0.5.0'>v0.5.0</a>.</em>

**Comment by seanbajwa** [comment]
Jack, is there a plug in I can run from the .txt files to convert/parse to readable formats in csv or pdf? Saw some updates from 2019/2020 documentation but wasn't following the logic

**Reply by jackmoody11** [reply]
Hey, Sean. You can use [BeautifulSoup](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#making-the-soup) to parse through it.

**Reply by seanbajwa** [reply]
thanks Jack, I like the file mapping use case
